### PR TITLE
Register and Tag PySyntax [https://github.com/kdheepak/PySyntax.jl.git]

### PR DESCRIPTION
**Pull request by kdheepak**

### CODE REVIEW

File: Py/versions/0.0.1/requires [hunk 1]
```diff
@@ -0,0 +1,2 @@
+julia 0.6
+PyCall 1.14.0
```
Comment (tkelman) [inline Py/versions/0.0.1/requires]: is this using a feature of 1.14 that doesn't work with any earlier versions?

**Comment by tkelman** [comment]
Should fill in the package description to something other than "No description, website, or topics provided," and enable Travis CI.

A more descriptive name might be a good idea as well.

File: Py/versions/0.0.1/requires (hunk 1 above)
Comment (kdheepak) [reply Py/versions/0.0.1/requires]: No, I don't believe so. It should be backwards compatible. I've made the change in the repo to remove the require for the specific PyCall version. I've also added julia 0.5 and 0.6 as compatible versions.

**Comment by kdheepak** [comment]
I've updated the package description. Do you think `PySyntax.jl` be a more descriptive name?

**Comment by kdheepak** [comment]
I changed the package name to `PySyntax.jl` and added a description to the repo.

**Comment by MikeInnes** [comment]
Thanks for the new package! You might consider announcing it on [Julia's forum](https://discourse.julialang.org/).

**Comment by kdheepak** [comment]
Thanks for the suggestion! I posted it on the forum, this will be a great way to get feedback.

**Pull request merged**